rspec-mode.el

```diff
@@ -761,7 +761,24 @@ file if it exists, or sensible defaults otherwise."
 
 (defun rspec--docker-default-wrapper (docker-command docker-container command)
   "Function for wrapping a command for execution inside a dockerized environment. "
-  (format "%s %s sh -c \"%s\"" docker-command docker-container command))
+    (if-let ((docker-cwd . ((rspec--docker-find-cwd))))
```

Comment:
This syntax is equivalent to:

```el
(if-let ((docker-cwd (rspec--docker-find-cwd)))
```

which should feel more natural to most Lisp programmers.